modules/openapi-generator/src/main/resources/ruby-client/api_client.mustache

```diff
@@ -255,6 +255,9 @@ module {{moduleName}}
     def build_request_url(path)
       # Add leading and trailing slashes to path
       path = "/#{path}".gsub(/\/+/, '/')
+      {{#compatibilitySlashesInPathParameters}}
+      path = path.gsub('%2F', '/')
```

Comment:
This needs to happen before `path = "/#{path}".gsub(/\/+/, '/')`, otherwise the URL starts with // when the path already started with /.

Reply:
Thank you.

I've amended the commit. The order is changed and `/%2F` going to `/` is now in the test cases.